edx_proctoring/models.py

```diff
@@ -256,6 +257,9 @@ class ProctoredExamReviewPolicy(TimeStampedModel):
     # policy that will be passed to reviewers
     review_policy = models.TextField()
 
+    # JSON rules that will be passed to reviewers
+    rules = JSONField(null=True)
```

Comment:
`null=true` because not all proctoring providers support JSON rules (e.g. RPNow).